### DISCUSSION TITLE
The document site's color gradient is so smooth? How to make a same one?

### DISCUSSION THREAD

**Discussion by adoin** [unocss/unocss#2490] [Q&A]
Or it's from other framwork?

**Comment by TomoyaKuroda** [comment]
The circle has a blur filter and linear gradient. The CSS animation changes the color of the linear gradient. Here is the demo.
https://codepen.io/tomoyanime/pen/eYPGXGo

**Comment by adoin** [comment]
Wow, thank you



---Original---
From: ***@***.***&gt;
Date: Wed, May 3, 2023 02:45 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [unocss/unocss] The document site's color gradient is so smooth?How to make a same one? (Discussion #2490)
